src/main/java/it/hospital/project/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package it.hospital.project.controller;
+
+import it.hospital.project.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
```

Comment:
If you import already four annotations from org.springframework.web.bind.annotation you can already use "*" import